<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>402</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="playerOneLayout">
    <item>
     <widget class="QLabel" name="playerOneNameLabel">
      <property name="text">
       <string>Player 1 Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="playerOneName"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Board Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="xSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ySize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="offline">
    <attribute name="title">
     <string>Singleplayer</string>
    </attribute>
    <widget class="QCheckBox" name="aiCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use AI</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="aiBeginsCheckbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AI Begins</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGameButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="playerTwoLayout">
      <item>
       <widget class="QLabel" name="playerTwoNameLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Player 2 Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="playerTwoName">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="online">
    <attribute name="title">
     <string>Multiplayer</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="ipAddressLayout">
      <item>
       <widget class="QLabel" name="ipAddressLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>IP Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipAddress">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hostServerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Host Server</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="highScoreButtonPressed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Highscores</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>xSize</zorder>
  <zorder>ySize</zorder>
  <zorder>label_5</zorder>
  <zorder>tabWidget</zorder>
  <zorder>highScoreButtonPressed</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
